bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-100to200/ZJetsToNuNu_HT100to200_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+import model sm-ckm_no_b_mass
+
+# Specify process(es) to run
+generate p p > ve ve~ / h @0
+add process p p > ve ve~ j / h @1
```

Comment:
is worth using all the neutrino flavors otherwise the cross section will be wrong by a factor 3

Reply:
You're right - do you know if this was fixed last time? I carried over the cards from e.g. here: https://github.com/cms-sw/genproductions/blob/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-1200to2500/ZJetsToNuNu_HT1200to2500_proc_card.dat

Reply:
From the Winter17 requests (e.g. https://cms-pdmv.cern.ch/mcm/requests?prepid=HIG-RunIIWinter15wmLHE-00766&page=0&shown=412585951359 )  I see that these cards were used: https://github.com/cms-sw/genproductions/blob/52fb4fa275013b316ef145a3677c02af4ff1975c/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-600to800/ZJetsToNuNu_HT600to800_proc_card.dat
Maybe the x-sec was scaled previously? I can of course update the cards (and reproduce the gridpacks), just let me know what the preferred solution is.

Reply:
one should not produce the gridpacks before the datacards are approved...